src/Corax/IndexWriter.cs

```diff
@@ -287,10 +301,6 @@ void AddMaybeAvoidDuplicate(List<long> term, long entryId)
                             var fieldValue = iterator.IsNull ? Constants.NullValueSlice : Constants.EmptyStringSlice;
                             ExactInsert(fieldValue.AsReadOnlySpan());
                         }
-                        else if ((fieldType & IndexEntryFieldType.Tuple) != 0)
```

Comment:
add a check here?

Reply:
We're handling tuple list before that. `Debug.Assert` maybe?